actions/ocr.go

```diff
@@ -269,6 +268,26 @@ func NewOCRSetupInputForAtlas(i *OCRSetupInputs, nodeCount int, contractCount in
 			i,
 			environment.NewChainlinkClusterForAtlasTesting(nodeCount),
 		))
+
+	err := i.SuiteSetup.Environment().DeploySpecs(environment.AtlasEvmBlocksGroup())
+	Expect(err).ShouldNot(HaveOccurred())
+
 	By("Funding nodes", FundNodes(i))
+
+	kafkaRestClient, err := environment.GetKafkaRestClientFromEnv(i.SuiteSetup.Environment())
+	Expect(err).ShouldNot(HaveOccurred())
+
+	Eventually(func(g Gomega) []string {
```

Comment:
I'm wondering if this is becoming a polluting test? Is this testing that OCR is working correctly? Or that Kafka connections and Atlas connections are working?

Reply:
This is here because of the order in which atlas-evm- : `events`, `receipts` and `blocks` are deployed.
First `blocks` has to be deployed because it creates 2 topics in kafka which are necessary for `events` and `receipts` to be deployed successfully. If this topics are not present, `events` and `topics` will fail to deploy